src/components/Library/LibraryForm/LibraryForm.tsx

```diff
@@ -13,30 +14,32 @@ export default function LibraryForm() {
     pages: '',
   };
 
+  const { t } = useTranslation();
+
   const formik = useFormik({
     initialValues,
     validationSchema: yup.object({
       name: yup
         .string()
-        .min(3, 'Min 3 symbols')
-        .max(50, 'Max 50 symbols')
-        .required('Required'),
+        .min(3, 'validation.min3')
+        .max(60, 'validation.max60')
+        .required('validation.required'),
       author: yup
         .string()
-        .min(3, 'Min 3 symbols')
-        .max(40, 'Max 40 symbols')
-        .required('Required'),
+        .min(3, 'validation.min3')
+        .max(40, 'validation.max40')
```

Comment:
maybe min 1 and max 100? names are very diferent

Reply:
Ok, will shange